KhronosGroup/Vulkan-LoaderAndValidationLayers
# Set up Infrastructure for and wire in first set of VUIDS for generated validation layers

- Modify parameter_validation codegen to compute VUID tags
- Use slightly modified vuid_mapping,py to convert tag to a VUID number
- Validate that vuid number against the existing set of good tags in `vk_validation_error_messages.h`
- Add VUID arg to the sType validation routine and calls
- Update error database

Since we do the check against the good messages, this works even though the spec may be incomplete -- when it does get updated the source will pick up the new VUIDS automatically.

This picks up around 70 VUs.
